backend/portfolio42_api/models.py

```diff
@@ -66,4 +72,50 @@ def __str__(self):
 		return f"{self.name}:{self.intra_id}"
 
 	def was_updated_today(self):
-		return self.updated_at > date.yesterday()
\ No newline at end of file
+		return self.updated_at > date.yesterday()
+
+## Relations
+# These are supposed to link the rest of the models together
+
+# This is a project that a user has subscribed to
+class ProjectUser(models.Model):
+	id_user = models.ForeignKey('User', on_delete=models.CASCADE)
+	id_project = models.ForeignKey('Project', on_delete=models.CASCADE)
+	intra_id = models.IntegerField(unique=True, db_index=True)
+	grade = models.IntegerField()
+	finished = models.BooleanField(default=False)
+	finished_at = models.DateTimeField()
+	updated_at = models.DateTimeField(auto_created=True, auto_now=True)
+
+	cursus = models.ManyToManyField('Cursus', through='ProjectUserCursus')
```

Comment:
We are going to remove this, the cursus should only be dependant on the Project, not the ProjectUser.